samples/subsys/audio/sof/boards/intel_adsp_cavs25.conf

```diff
@@ -5,3 +5,9 @@ CONFIG_INTEL_ALH=y
 CONFIG_LP_MEMORY_BANKS=1
 CONFIG_HP_MEMORY_BANKS=30
 CONFIG_RIMAGE_SIGNING_SCHEMA="tgl"
+CONFIG_PM=y
+CONFIG_PM_DEVICE=y
+CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_POWER_DOMAIN=n
+CONFIG_POWER_DOMAIN=n
+CONFIG_POWER_DOMAIN_INTEL_ADSP=n
```

Comment:
Not sure about this, but wondering if the change to cavs25 board should be in separate commit (but part of this PR). 

Also, why CONFIG_POWER_DOMAIN=n and CONFIG_POWER_DOMAIN_INTEL_ADSP=n in this patch?

Reply:
Yeah, probably should have dropped unused configs. Power domains are off, as there are no power domain definitions for cavs. I'm not targeting cavs platform at the moment and have ace definitions prepared.

Reply:
Ack @kfrydryx . That could be a problem for merge if there is not user in the tree for the option. So you'd need either use cAVS2.5, add a test case or wait for the ACE board file to be added. @ceolin any guidance here?

Reply:
First, these options don't need to be set, these are the default value.

Regarding not having a use in tree, that is problematic indeed, since there is not test at all, we are not even building it.  My suggestion is to wait the board be added. We can work in the changes but we should have a way to build it before merging.